vendor/k8s.io/kubernetes/pkg/kubectl/cmd/get.go

```diff
@@ -299,6 +300,11 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		if err := printer.PrintObj(obj, out); err != nil {
 			allErrs = append(allErrs, err)
 		}
+
+		// output empty list warning when no resources to display
+		if len(infos) == 0 {
+			fmt.Fprintf(os.Stderr, "%s\n", "There are no resources to display.")
```

Comment:
Do we refer to them as 'filters' in the flag descriptions? If so I would probably say this instead.
```
No resources found in namespace "mynamespace" that match your filters.
```

If the only flag that filters is selector then i'd say

```
No resources found in namespace "mynamespace" that matches the selector.
```

But I would only be that specific if we know they used the flag.  If this output is in a generic place then be generic and go with criteria or filters.